README.md

```diff
@@ -1,19 +1,186 @@
+## Persona: Designer
+
+For the designer writing the document,
+Duvet is a mechanism to get requirements implemented and tested.
+You sprinkle in `MUST` or `SHOULD` into your document,
+and you can block the build until every requirement
+not only has an implementation, but also a test.
+
+![](./img/designer.png)
+
+## Persona: Developer
+
+For the developer writing the implementation
+Duvet maintains the punch list for you,
+you can focus on getting the best implementation for a given requirement.
+As you write the code you can ask:
+Dose this code satisfy this requirement?
+Does this test verify this behavior?
+If you need details,
+the document is right there with delicious cake-like background.
+
+![](./img/developer.png)
+
+## Persona: Reviewer
+
+For the reviewer reading the code
+Duvet turns the design document into a map of the code.
+As you review the code the details are right there for you.
+You can jump right to the test and really focus on
+"Does this test convince me that the requirement MUST be met?"
+
+The reviewer can also updates the design.
+Running duvet on the new document
+the developer is now on the hook
+to both implement and test these new requirements.
 
-## Getting Started
+![](./img/reviewer.png)
+
+## Persona: Debugger  consumer?
+
+For the debugger coming to back to a project
+that they have not worked on in a long time,
+or are completely new to
+Duvet annotations create a network effect.
+These annotations help give context to code
+especially if the code is not your preferred language.
+They link back to the design document
+to give background and more information.
+Since these annotations are bidirectional,
+are waypoints that let you move between
+the more abstract design document
+and the detailed implementation.
+
+![](./img/debugger.png)
+
+## When do you stop adding tests?
+
+But now we are done right?
+Every requirement has an implementations and a tests.
+They all have been reviewed.
+Duvet can only see that you TRIED to test.
+How does the reviewer know you tested ENOUGH?
+
+Duvet is a mechanism to connect the requirement to the source,
+but it leaves the quality and completeness of these links to humans.
+A tests makes sure that something is true once,
+it does not make sure that something is *always* true.
+Todays edge case it tomorrow's issue.
+
+## Dafny
+
+Dafny is a *verification-aware* programming language.
+
+Formal verification let's us translate the Duvet annotation
+into a specification that says
+"for every string in the *universe* this must **always** be true".
+The Dafny verifier will use deductive reasoning to say "Yup, that is true".
+This is like writing a test, but you are testing every value all at once
+statically before you run any code.
+
+Dafny uses formal logic to prove specifications.
+It doesn't forget about edge cases
+because it checks every case.
+
+Since is very useful to have verified code,
+but only if you can run it
+Dafny compiles to many other languages:
```

Comment:
```suggestion
Since is useful to have verified code,
but much more useful if you can run it,
Dafny compiles to many other languages:
```

Just to avoid implying that modeling and proving some other code over there is not useful. :)